executor/fktest/BUILD.bazel

```diff
@@ -10,9 +10,12 @@ go_test(
     flaky = True,
     deps = [
         "//config",
+        "//executor",
         "//meta/autoid",
+        "//parser/model",
         "//planner/core",
         "//testkit",
+        "//types",
```

Comment:
Here are some extra changes, because someone forgot to update the bazel rules (through `make bazel_prepare`). I have checked with @hawkingrei and I could submit these diffs with this PR.